static/js/rest.js

```diff
@@ -102,13 +107,19 @@ export const endpoints: Array<Endpoint> = [
   {
     name: 'payment',
     url: '/api/v0/payment/',
-    makeOptions: (total, klassId) => ({
+    fetchOptions: (params) => ({
```

Comment:
was there a specific reason for this change? personally I think we may want to stick with the rest parameter (for the argument passed to the function returned by `makeAction`) for flexibility but this would work equally well, just wondering what your thinking was here.